code/game/machinery/doors/airlock.dm

```diff
@@ -1293,7 +1293,7 @@
 			to_chat(user, span_warning("The airlock won't budge!"))
 			return
 
-		var/time_to_open = 7 SECONDS * I.toolspeed
+		var/time_to_open = 9 SECONDS
```

Comment:
Why do we not want toolspeed to be taken into account?

Reply:
It's already taken into account in use_tool()